proxy/src/main/java/org/dragonet/proxy/network/session/cache/ChunkCache.java

```diff
@@ -80,7 +80,11 @@ public ChunkData translateChunk(ProxySession session, int columnX, int columnZ)
 
             List<CompoundTag> bedrockBlockEntities = new ArrayList<>();
             for(int i = 0; i < column.getTileEntities().length; i++) {
-                CompoundTag tag = BlockEntityTranslatorRegistry.translateToBedrock(column.getTileEntities()[i]);
+                int x = Integer.parseInt(column.getTileEntities()[i].get("x").getValue().toString());
```

Comment:
You can also cast it to an IntTag rather than returning it as a string and parsing it